### PR TITLE
Update README.md with example tab mapping

### LINKED ISSUE #141: how to setup the tab completion?
can you help me pls, coz I'm new to Neovim,
like I want after trigger of completion I want to navigate through tab and S-tab instead of C-n and C-p

### PR DESCRIPTION
**Pull request by cloggier**
Closes #141 

This is what I am using, perhaps it can be improved somehow?

### CODE REVIEW

**Comment by cloggier** [comment]
Added some other changes to readme. Formatted with prettier and run through linter.

File: README.md [hunk 1]
```diff
@@ -185,26 +191,64 @@ inoremap <silent><expr> <C-e>     compe#close('<C-e>')
...
-```
+   require'lspconfig'.rust_analyzer.setup {
+     capabilities = capabilities,
+   }
+   ```
 
 2. Install `vim-vsnip`
 
-`Plug 'hrsh7th/vim-vsnip'`
+   `Plug 'hrsh7th/vim-vsnip'`
```
Comment (ii14) [inline README.md:211]: Inconsistent formatting, add triple backticks here

File: README.md [hunk 2]
```diff
@@ -185,26 +191,64 @@ inoremap <silent><expr> <C-e>     compe#close('<C-e>')
...
+  if vim.fn.pumvisible() == 1 then
+    return t "<C-p>"
+  elseif vim.fn.call("vsnip#jumpable", {-1}) == 1 then
+    return t "<Plug>(vsnip-jump-prev)"
+  else
+    return t "<S-Tab>"
+  end
+end
+
+map("i", "<Tab>", "v:lua.tab_complete()", {expr = true})
```
Comment (hrsh7th) [inline README.md:247]: Please use Vim style mapping (or add `map` function here

Lua style mappings are bit annoying... Hmm...(due to 
 `vim.api.nvim_replace_termcodes`

**Comment by hrsh7th** [comment]
Thank you for enhancing docs. It will save issue handling cost.

File: README.md (hunk 1 above)
Comment (cloggier) [reply README.md]: Good point. Changed. markdownlint did not report this one.

File: README.md (hunk 2 above)
Comment (cloggier) [reply README.md]: Ah, missed that, sorry. Added full api call. Is that OK or would you prefer to have shortcut like `local map = vim.api.nvim_set_keymap`?

Comment (hrsh7th) [reply README.md]: Full api is good. Thank you!

**Comment by hrsh7th** [comment]
LGTM. Thank you.

**Pull request merged**